onnx_tf/handlers/frontend/control_flow_common.py

```diff
@@ -0,0 +1,30 @@
+from onnx_tf.common.broadcast import get_broadcast_axis
+from onnx_tf.handlers.frontend_handler import FrontendHandler
+
+
+class LogicalCommon(FrontendHandler):
+
+  @classmethod
+  def logical_op(cls, node, broadcast=1, **kwargs):
```

Comment:
It seems like you did you own categorization of ops. In my opinion, a **Taxonomy** (i.e., classification) of ops is very difficult to come up with from scratch and even more difficult to justify and remember. Thus I would recommend that we follow the ONNX's way of categorizing them to remove our intellectual burden of maintaining yet another hierarchical classification of operations. Specifically, `Less Op` is a Logical Op, not a `Comparison Op`  in ONNX (c.f. https://github.com/onnx/onnx/blob/6f4b1b12e56cea736a31411c5f151305f211a729/onnx/defs/logical/defs.cc#L68).

Reply:
@tjingrant 
I am following tf,
https://www.tensorflow.org/api_guides/python/math_ops
https://www.tensorflow.org/api_guides/python/control_flow_ops
of course, follow onnx is a good idea also.

Reply:
I think following TF is probably fine. Let's keep it this way then.

Reply:
BTW, can we document that here https://github.com/onnx/onnx-tensorflow/blob/master/onnx_tf/doc/IMPLEMENTING_NEW_OP.md ? We need an update there anyways.